<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAjouterRessource</class>
 <widget class="QDialog" name="DialogAjouterRessource">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>AJOUTER UN PERSONNEL</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pr√©nom*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Type*</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>281</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Indisponsable pour les informaticiens !</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>223</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>45</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Pseudo*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_pseudo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Mot de passe*</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_mdp">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nom">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_prenom">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>151</y>
      <width>165</width>
      <height>216</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Banquier A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_banquierA">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Banquier B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_banquierB">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Assureur logement</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_assureurLogement">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Assureur voiture</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_assureurVoiture">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Assureur vie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_assureurVie">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Divers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_divers">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Informaticien </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_informaticien">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>560</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_annuler">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_nom</tabstop>
  <tabstop>lineEdit_prenom</tabstop>
  <tabstop>radioButton_banquierA</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>radioButton_banquierB</tabstop>
  <tabstop>radioButton_assureurLogement</tabstop>
  <tabstop>radioButton_assureurVoiture</tabstop>
  <tabstop>radioButton_assureurVie</tabstop>
  <tabstop>radioButton_informaticien</tabstop>
  <tabstop>lineEdit_pseudo</tabstop>
  <tabstop>lineEdit_mdp</tabstop>
  <tabstop>pushButton_annuler</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_banquierA</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_banquierB</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_assureurLogement</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_assureurVoiture</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_assureurVie</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_informaticien</sender>
   <signal>clicked()</signal>
   <receiver>groupBox_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
